src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -974,6 +968,10 @@ export class MeetingManager implements AudioVideoObserver {
       callback(name, attributes)
     );
   };
+
+  get eventController(): EventController {
```

Comment:
Why does this have to be a get API? Do we access it anywhere else?

Reply:
I use it in the unit test to publish events

Reply:
Can you just create your own event controller and pass it into meeting provider/manager? We probably don't want to add API just for unit test.

Reply:
Ah yeah that works, I didn't think of that